docs/source/documentation/concepts.rst

```diff
@@ -160,6 +160,31 @@ There are two ways to run several tasks in parallel on a same organization with
 .. TODO:: Explain what is substra tools
 
 
+Substra channels
+----------------
+
+A channel is a group of Substra `organizations <https://docs.substra.org/en/stable/additional/glossary.html#term-Organization>`_ which operates on a common set of assets.
+
+Having multiple channels is needed if you want to have multiple independent projects on the same network. 
+
+In the scheme below, we represent a Substra network with two channels:
+
+- Channel A with the three organizations
+- Channel B with only two organizations (A and B)
+
+.. image:: /static/schemes/channel_schema.png
+
+Users only belong to one channel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A user can only belong to one channel. A channel has its own users, its own datasets, compute plans, tasks and functions. A user from another channel has no visibility on this. A user has no visibility on a channel they are not a part of.
+
+What happens if an organization gets multiple tasks from different channels?
```

Comment:
I don't have anything against this title per se, it's fine at describing its contents. However it doesn't fit with the other titles, which aren't sentences.

This is a very minor remark, feel free to keep as-is